dotnet/src/Connectors/Connectors.AssemblyAI/AssemblyAIClientFactory.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using AssemblyAI;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel.Connectors.AssemblyAI;
+
+internal static class AssemblyAIClientFactory
```

Comment:
Do you really need this class? Can you move this logic closer usage?

Reply:
I moved it inline